Azure_Pipelines/azure-pipelines.terminology.cd.yaml

```
@@ -0,0 +1,100 @@
+    - DotNet/Terminology/*
+    - DotNet/Shared/*
+    exclude:
+    - '*'
+    
+pr: none
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+variables:
+- group: link-cloud-variables
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 2 but found 0

(indentation)